CMakeLists.txt

```diff
@@ -484,7 +483,11 @@ endif()
 # "bluevk" and "samples" targets.
 # ==================================================================================================
 
-if (FILAMENT_SUPPORTS_VULKAN AND APPLE)
+if (FILAMENT_USE_SWIFTSHADER)
```

Comment:
Should we verify that `FILAMENT_SUPPORTS_VULKAN` is set here? SwiftShader is only useful with Vulkan support, correct?

Reply:
Good idea, added an error message here.